### PR TITLE
Split lodash

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

**Comment by phated** [comment]
Instead of the `_` namespace, can you just update the references?

File: lib/helpers.js [hunk 1]
```diff
@@ -40,14 +45,14 @@ function buildOnSettled(done) {
...
   var lastIdx = args.length - 1;
 
   assert.ok(args.length, 'A set of functions to combine is required');
 
-  _.forEach(args, function(arg, argIdx) {
-    var isFunction = _.isFunction(arg);
-    if (isFunction) {
+  forEach(args, function(arg, argIdx) {
+    var isFunctionArg = isFunction(arg);
+    if (isFunctionArg) {
```
Comment (TrySound) [inline lib/helpers.js]: Maybe better use just `typeof arg === 'function'`?

Comment (TrySound) [inline lib/helpers.js]: One extra dependency less :)

Comment (phated) [inline lib/helpers.js]: sounds good. might as well drop noop then too

File: lib/helpers.js [hunk 2]
```diff
@@ -2,7 +2,12 @@
 
 var assert = require('assert');
 
-var _ = require('lodash');
+var noop = require('lodash.noop');
+var filter = require('lodash.filter');
+var map = require('lodash.map');
+var flatten = require('lodash.flatten');
+var forEach = require('lodash.foreach');
```
Comment (TrySound) [inline lib/helpers.js]: This foreach do nothing more native. Maybe drop it too?

Comment (phated) [inline lib/helpers.js]: I'd like this library to work in es3, so I think it should stay

**Comment by TrySound** [comment]
Sorry for travis noise. I'm on windows. jscs do not support globs (only via bash) and linebreaks rule sucks. We removed it in postcss shared config. Better to use `.gitattributes`

File: lib/helpers.js [hunk 3]
```diff
@@ -12,7 +15,7 @@ function getExtensions(lastArg) {
 
 function buildOnSettled(done) {
   if (typeof done !== 'function') {
-    done = _.noop;
+    done = function() {};
```
Comment (phated) [inline lib/helpers.js]: hoist noop and make it a named function below the requires

**Comment by TrySound** [comment]
@phated Done.

**Comment by phated** [comment]
Published as 0.5.0

**Pull request merged**